src/status_im2/constants.cljs

```diff
@@ -334,3 +334,29 @@
 (def ^:const auth-method-none "none")
 
 (def ^:const image-description-in-lightbox? false)
+(def ^:const int->weekday
```

Comment:
this should be in time utils, not here

Reply:
[Follow up PR](https://github.com/status-im/status-mobile/pull/16350)